deis.go

```diff
@@ -89,7 +89,7 @@ type Client struct {
 // controller is unsafe.
 //
 // If the SDK detects an API version mismatch, it will return ErrAPIMismatch.
-const APIVersion = "2.1"
+const APIVersion = "2.2"
```

Comment:
However this is stuck until all the PRs go green though so we don't have an API version mismatch between the client and the server.

I can PR this separately if you'd like!